server/src/main/java/org/opensearch/index/store/lockmanager/RemoteStoreMetadataLockManager.java

```diff
@@ -88,6 +89,11 @@ public String fetchLock(String filenamePrefix, String acquirerId) throws IOExcep
         return lockFilesForAcquirer.get(0);
     }
 
+    public Set<String> fetchLocks(String filenamePrefix) throws IOException {
```

Comment:
nit - could this be renamed to fetchLockedFiles rather to suggest it return names of the files (& not locks)

Reply:
Had the same comment earlier by @gbbafna 

Let me make the change.